subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -1385,8 +1385,8 @@ uint8_t ull_adv_aux_hdr_set_clear(struct ll_adv_set *adv,
 {
 	struct pdu_adv_com_ext_adv *pri_com_hdr, *pri_com_hdr_prev;
 	struct pdu_adv_com_ext_adv *sec_com_hdr, *sec_com_hdr_prev;
-	struct pdu_adv_ext_hdr *pri_hdr, pri_hdr_prev;
-	struct pdu_adv_ext_hdr *sec_hdr, sec_hdr_prev;
+	struct pdu_adv_ext_hdr *hdr, pri_hdr, pri_hdr_prev;
```

Comment:
With so many `hdr` variables in this function, just using `hdr` seems a bit too generic. 

Would it make sense to call it something like `ext_hdr`?

Reply:
Ah, so `hdr` is used in multiple different contexts, and does not refer to a specific type of header? In that case I think it is OK to keep as is.

Reply:
yes, `hdr` is used in multiple context temporarily.